src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/index.tsx

```diff
@@ -377,8 +394,25 @@ const MissingItemFormEdit = () => {
                 </Grid>
               </Grid>
 
-              <Grid container justifyContent="flex-end" className={styles.submit_container}>
-                <Grid item xs={2}>
+              <Grid
+                container
+                spacing={2}
+                justifyContent="center"
+                className={styles.submit_container}
+              >
+                {formData.status.toLowerCase() === 'found' && (
```

Comment:
I'm thinking maybe something like this on the edit page.  Gives the instructions again, and with more details this time.  And the button to mark as picked up is very obvious!  I think this should make it more clear what to do to clear the notification:
<img width="912" alt="Screen Shot 2024-12-03 at 23 07 05" src="https://github.com/user-attachments/assets/03442a50-f21a-435a-8cac-322d5da192b2">


Button design after being clicked:
<img width="385" alt="Screen Shot 2024-12-03 at 23 09 24" src="https://github.com/user-attachments/assets/0e7f58c5-a545-4c01-a697-45224f65d48a">

Reply:
Yh sure, looks fine

Reply:
I think the design here doesn't look great, but the content of it is what I think should be there.  Whether there's a better way to design how it looks, idk, but this is good enough probably.